recipes/erlang/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "18.3" %}
+
+package:
+  name: erlang
+  version: {{ version }}
+
+source:
+  fn: otp_src_{{ version }}.tar.gz
+  url: http://erlang.org/download/otp_src_{{ version }}.tar.gz
+  sha256: "fdab8129a1cb935db09f1832e3a7d511a4aeb2b9bb3602ca6a7ccb9730d5c9c3"
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - m4
+    - perl
+    - readline 6.2*
+    - openssl 1.0.*
+    - ncurses
+    - zlib 1.2*
+    - libatomic_ops
+  run:
+    - readline 6.2*
+    - openssl 1.0.*
+    - ncurses
+    - zlib 1.2*
+    - libatomic_ops
+
+test:
+  commands:
+    - erl -version
+
+about:
+  home: http://www.erlang.org/
+  license: Erlang Public License 1.1 (http://www.erlang.org/EPLICENSE)
```

Comment:
Appears this is now Apache 2.0, which happened with the 18.0 release. See this issue ( https://github.com/conda-forge/erlang-feedstock/issues/2 ).